cvat/apps/engine/serializers.py

```diff
@@ -807,20 +996,21 @@ def validate(self, attrs):
 
 
 class ProjectReadSerializer(serializers.ModelSerializer):
-    labels = LabelSerializer(many=True, source='label_set', partial=True, default=[], read_only=True)
     owner = BasicUserSerializer(required=False, read_only=True)
     assignee = BasicUserSerializer(allow_null=True, required=False, read_only=True)
     task_subsets = serializers.ListField(child=serializers.CharField(), required=False, read_only=True)
     dimension = serializers.CharField(max_length=16, required=False, read_only=True, allow_null=True)
     target_storage = StorageSerializer(required=False, allow_null=True, read_only=True)
     source_storage = StorageSerializer(required=False, allow_null=True, read_only=True)
-    tasks = HyperlinkedModelViewSerializer(models.Task, filter_key='project_id')
+    tasks = CollectionSummarySerializer(models.Task, url_filter_key='project_id')
```

Comment:
Maybe need also to define `TasksSummarySerializer` here and in other places where `CollectionSummarySerializer` is used. (just for visibility)
![image](https://user-images.githubusercontent.com/49038720/219665917-b3b2ff51-3a54-4d04-9e76-f435f75f9ad9.png)

Reply:
Done